Update Exchange-Teams-interact.md (#7540)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/Exchange-Teams-interact.md

Users with mailboxes hosted on-premises must be synchronized to Azure Active Directory. They can make use of all the features in the above scenario, but additionally, they can manage meetings if the requirements listed on [Requirements for mailboxes hosted on-premises](#requirements-to-create-and-view-meetings-for-mailboxes-hosted-on-premises) section are met.

The following table provides a helpful quick reference to feature availability based on the Exchange environment.
 > [!NOTE]

>  > Profile picture change is not supported for shared mailboxes.

Reviewer: ```suggestion
 > Modifying profile pictures is not supported on shared mailboxes.
 
```
